utoipa-gen/src/component/schema.rs

```diff
@@ -278,6 +278,30 @@ impl NamedStructSchema<'_> {
             .parse_features::<NamedFieldFeatures>()
             .into_inner();
 
+        let schema_default = self
+            .features
+            .as_ref()
+            .map(|features| features.iter().any(|f| matches!(f, Feature::Default(_))))
+            .unwrap_or(false);
+        let serde_default = serde::parse_container(self.attributes)
+            .map(|rules| rules.default)
+            .unwrap_or(false);
```

Comment:
We probably could pass the container args to the function so we do not need to parse it again here. From this line: https://github.com/juhaku/utoipa/pull/533/files#diff-f44986f2c073c8469bf9086f7d719014eacc7ae5dee63186c941cec5c8dd5e22L340